component/client-handler/src/test/resources/dbscripts/identity.sql

```diff
@@ -7,21 +7,22 @@ INSERT INTO IDN_BASE_TABLE values ('WSO2 Identity Server');
 
 CREATE TABLE IF NOT EXISTS IDN_OAUTH_CONSUMER_APPS (
             ID INTEGER NOT NULL AUTO_INCREMENT,
-            CONSUMER_KEY VARCHAR (255),
-            CONSUMER_SECRET VARCHAR (512),
-            USERNAME VARCHAR (255),
+            CONSUMER_KEY VARCHAR(255),
+            CONSUMER_SECRET VARCHAR(2048),
+            USERNAME VARCHAR(255),
             TENANT_ID INTEGER DEFAULT 0,
             USER_DOMAIN VARCHAR(50),
-            APP_NAME VARCHAR (255),
-            OAUTH_VERSION VARCHAR (128),
-            CALLBACK_URL VARCHAR (1024),
+            APP_NAME VARCHAR(255),
+            OAUTH_VERSION VARCHAR(128),
+            CALLBACK_URL VARCHAR(1024),
             GRANT_TYPES VARCHAR (1024),
             PKCE_MANDATORY CHAR(1) DEFAULT '0',
             PKCE_SUPPORT_PLAIN CHAR(1) DEFAULT '0',
             APP_STATE VARCHAR (25) DEFAULT 'ACTIVE',
-            USER_ACCESS_TOKEN_EXPIRE_TIME BIGINT DEFAULT 3600000,
-            APP_ACCESS_TOKEN_EXPIRE_TIME BIGINT DEFAULT 3600000,
-            REFRESH_TOKEN_EXPIRE_TIME BIGINT DEFAULT 84600000,
+            USER_ACCESS_TOKEN_EXPIRE_TIME BIGINT DEFAULT 3600,
```

Comment:
We are changing the default value from 3600000 -> 3600. Is this correct?

Reply:
This is the same table structure that was used in IS-5.8.0